markdown/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
@schalkneethling Another question is about the changes to `markdown/*` and `markdown/m2h`. If I remember correctly, you extracted `@mdn/markdown-converter` to a separate repository, removing it from `yari`, but then following an exchange with Florian, you reverted that?

I'm also wondering whether this `tsconfig.json` got lost intentionally, or accidentally as part of the removal/restore?

Reply:
Intentionally. Everything was converted to JS so there should not be any TS anymore in the markdown app.